Classes/Projection/ProjectionManager.php

```diff
@@ -145,24 +145,34 @@ public function replay(string $projectionIdentifier)
         return $eventCount;
     }
 
-//    /**
-//     * @param string $projectionIdentifier
-//     * @return void
-//     */
-//    public function catchup(string $projectionIdentifier)
-//    {
-//        $fullProjectionIdentifier = $this->normalizeProjectionIdentifier($projectionIdentifier);
-//        $cacheId = md5($fullProjectionIdentifier);
-//        if (!$this->projectionCache->has($cacheId)) {
-//            $projectionState = [
-//                'revision' => 0
-//            ];
-//        } else {
-//            $projectionState = $this->projectionCache->get($cacheId);
-//            $projectionState['revision']++;
-//        }
-//        $this->projectionCache->set($cacheId, $projectionState);
-//    }
+    /**
+     * @param string $projectionIdentifier
+     * @return int Number of events which have been applied
+     */
+    public function catchup(string $projectionIdentifier): int
```

Comment:
in English there's no word "catchup", so we better call it "catchUp", or?
Unless you mean Ketchup, of course ;-)